cpanfile

```diff
@@ -1,6 +1,10 @@
 # kind of duplicate of Makefile.PL
 #	but convenient for Continuous Integration
 
+on 'build' => sub {
+    requires 'Test::Glob' => 0;
```

Comment:
you probably mean `Text::Glob` here
without it, CI is broken

Reply:
Oh crap. Yes.

Reply:
Force-pushed fix.